pkg/virtctl/credentials/addkey/addkey.go

```diff
@@ -10,7 +10,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/tools/clientcmd"
-	"k8s.io/utils/pointer"
+	"k8s.io/utils/ptr"
```

Comment:
Please use "kubevirt.io/kubevirt/pkg/pointer". This dependency is redundant.

Reply:
done